src/scripts/main.js

```diff
@@ -1 +1,86 @@
 'use strict';
+
+const body = document.querySelector('body');
+body.addEventListener('contextmenu', (event) => {
```

Comment:
Could you fix these tests? If you have some questions - feel free to ask them on `fe_chat`
![image](https://user-images.githubusercontent.com/56752562/222663883-191ac71d-95cf-4c17-9f69-75c67aaa8ce9.png)